src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/Microsoft.VisualStudio.Web.CodeGeneration.Tools.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\dir.props" />
+  <PropertyGroup>
+    <Description>Code Generation tool for ASP.NET Core. Contains the dotnet-aspnet-codegenerator command used for generating controllers and views. </Description>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <AssemblyName>dotnet-aspnet-codegenerator</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageTags>aspnetcore;codegenerator;scaffolding;visualstudioweb</PackageTags>
+    <PackageVersion>$(VersionPrefix)-$(VersionSuffix)</PackageVersion>
+    <PackageId>Microsoft.VisualStudio.Web.CodeGeneration.Tools</PackageId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="**\*.cs" Exclude="bin\**;obj\**" />
+    <Compile Include="..\Shared\**\*.cs" Exclude="bin\**;obj\**" />
+    <EmbeddedResource Include="**\*.resx" />
+    <EmbeddedResource Include="Compiler\Resources\**\*" />
```

Comment:
Nit: might cause issues with case-sensitivity on linux. It seems other projects in this repo have this path lowercased.